rest_framework/permissions.py

```diff
@@ -122,6 +122,9 @@ def has_permission(self, request, view):
 
         if hasattr(view, 'get_queryset'):
             queryset = view.get_queryset()
+            assert queryset is not None, (
+                'The `.get_queryset()` method from the view did not return anything.'
```

Comment:
Can we give a more informative message here? 

Perhaps:

    'Return of {}.get_queryset() was None'.format(view.__class__.__name__)